web/src/components/AttributeList/AttributeList.tsx

```diff
@@ -21,11 +20,15 @@ const AttributeList: React.FC<IProps> = ({assertions, attributeList, onCreateAss
     navigator.clipboard.writeText(value);
     setIsCopied(true);
   };
+  const [isAnyHovered, setIsAnyHovered] = useState<number[]>([]);
```

Comment:
I don't like how this seems to be very tightly coupled with the rest of the component structure

Reply:
Yes it is very coupled with the structure.
What do you suggest then? @xoscar @jorgeepc